### PR TITLE
doc: mention BIP65 softfork in bips.md

### PR DESCRIPTION
**Pull request by laanwj**
Add a line for BIP65

### CODE REVIEW

**Comment by btcdrak** [comment]
Proof-read. ACK

**Comment by PRabahy** [comment]
Link/text doesn't match for mempool part. ([PR #6351](https://github.com/bitcoin/bitcoin/pull/6124))

File: doc/bips.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.12.0**):
 * [`BIP 37`](https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki): The bloom filtering for transaction relaying, partial merkle trees for blocks, and the protocol version bump to 70001 (enabling low-bandwidth SPV clients) has been implemented since **v0.8.0** ([PR #1795](https://github.com/bitcoin/bitcoin/pull/1795)).
 * [`BIP 42`](https://github.com/bitcoin/bips/blob/master/bip-0042.mediawiki): The bug that would have caused the subsidy schedule to resume after block 13440000 was fixed in **v0.9.2** ([PR #3842](https://github.com/bitcoin/bitcoin/pull/3842)).
 * [`BIP 61`](https://github.com/bitcoin/bips/blob/master/bip-0061.mediawiki): The 'reject' protocol message (and the protocol version bump to 70002) was added in **v0.9.0** ([PR #3185](https://github.com/bitcoin/bitcoin/pull/3185)).
+* [`BIP 65`](https://github.com/bitcoin/bips/blob/master/bip-0065.mediawiki): The CHECKLOCKTIMEVERIFY softfork was merged in **v0.12.0** ([PR #6351](https://github.com/bitcoin/bitcoin/pull/6351)), and backported to **v0.11.2** and **v0.10.4**. Mempool-only CLTV was added in ([PR #6351](https://github.com/bitcoin/bitcoin/pull/6124)).
```
Comment (btcdrak) [inline doc/bips.md]: PR #6351 links to https://github.com/bitcoin/bitcoin/pull/6124

Comment (laanwj) [inline doc/bips.md]: Yes, should be 6124, otherwise it's twice 6351, good catch

**Comment by petertodd** [comment]
ACK modulo fixing mismatched text.

**Comment by fanquake** [comment]
ACK post link fix

**Comment by laanwj** [comment]
Fixed the link name

**Pull request merged**